### ISSUE TITLE
Metrics contain realm uuid instance of name with KC19

### ISSUE THREAD

**Issue by diversit** [aerogear/keycloak-metrics-spi#143]
## Description
Adding the metrics-spi to KC19 creates metrics with a UUID value for the 'real' label.

## Expected Behavior
The 'real' label should contain the realm name.

## Actual Behavior
The 'real' label contains a UUID.

## Environment

* Project Versions: 
  * keycloak-metrics-spi : 2.5.3
  * keycloak: 19.0.1

## Steps to reproduce
- Create custom keycloak 19 image with keycloak-metrics-spi jar
- Run container
- Enable metrics-spi Events listener
- Login to Keycloak
- View metrics : curl http://<url>/auth/realms/master/metrics

**Comment by pb82** [comment]
fixed in #159

**Comment by souravs17031999** [comment]
@pb82 Are you sure this is fixed ? I tried with v3.0.0 with KC 21 and it's still showing UUID as shown 
```
Received user event of type LOGIN in realm e1803681-464d-447a-85a1-12a57a111da5
```
